### ISSUE TITLE
Typography

### ISSUE THREAD

**Issue by jasonkliu** [YaleSTC/YalePublic-android#47]
On the Start screen of the app, it should be changed to the correct logo and typography. 

NOTE: Fonts should not be publicly distributed so they should be put in the private repo or an alternate solution should be found.

http://www.yale.edu/printer/typeface/
http://identity.yale.edu/

**Comment by krysjez** [comment]
Do you mean a splash screen?
We can just use an image, get a transparent/white Yale logo from the identity site and center it on a blue screen.

**Comment by jasonkliu** [comment]
I believe @clhpeterson already implemented the splash screen.

**Comment by Ebstein** [comment]
@jasonkliu @krysjez I changed the font sizes within the videos and changed the appearance a little bit so that the text is bigger as you guys wanted. Did not do typography yet.

**Comment by Ebstein** [comment]
@jasonkliu @krysjez Please do not touch the pull request yet. I have started doing the typefonts. I can do it 1 by 1 in the textviews but it is very tedious. I have tried to use this aproach to change them all at once:

http://stackoverflow.com/questions/2711858/is-it-possible-to-set-font-for-entire-application

But its not compatible with v. 21 and the theme we are using. I'll try to make it work some time later today

**Comment by Ebstein** [comment]
there exist workarounds but nothing that is actually good enough. Hence, I will just do it all as a case-by-case basis.

**Comment by Ebstein** [comment]
Not done yet... the actionbars have to be rewritten in a custom way...

**Comment by krysjez** [comment]
[Just finished my midterm paper!!!! That was way too hard]

sorry, quick question, what are the typefonts?
Aren't we using the default Roboto on everything?

**Comment by jasonkliu** [comment]
Can't you define a custom style for a TextView (with size, font, etc) and set all to the same style?
Like this - https://developer.android.com/guide/topics/ui/themes.html?

**Comment by Ebstein** [comment]
Nope. That only works with the built-in typefonts as far as I have
researched

**Comment by krysjez** [comment]
@ebstein are you changing the fonts on the menu screen? Aren't those already done?
